packages/shared/validators.js

```diff
@@ -76,6 +83,20 @@ export function isNameSelector (nameOptionsObject: any): boolean {
   return !!nameOptionsObject.name
 }
 
+export function isTagSelector (selector: any) {
+  if (typeof selector !== 'string') {
+    return false
+  }
+  const pseudoSelectors = ['.', '#', '[', ':', '>', ' ']
+  if (htmlTags.includes(selector) ||
+    svgElements.includes(selector) ||
+    selector.split('').some(char => pseudoSelectors.includes(char))) {
```

Comment:
I think we should throw an error if the user is using a component selector with a pseudo selector, because they might expect this to work and be confused by find returning false.